src/app/exact-currency.pipe.ts

```diff
@@ -27,7 +27,7 @@ export class ExactCurrencyPipe implements PipeTransform {
     // Donor inputs get passed to this pipe directly, so we need to ensure non-numeric
     // input can't crash the app.
     value = Number(value);
-    if (isNaN(value)) {
+    if (! Number.isFinite(value)) {
```

Comment:
not particularly related to the current task but thought we might as well return undefined for infinity as well as for NaN.